projects/go-lib/src/lib/components/go-toast/go-toast.component.scss

```diff
@@ -15,6 +15,11 @@
     justify-content: flex-end;
     padding: .875rem 1.25rem;
   }
+
+  &--dark {
+    background-color: $theme-dark-bg;
+    color: $theme-dark-color;
+  }
```

Comment:
If we do this here, we can get rid of the extra `ngClass`es in the template around the `go-toast-content` class and remove the `&--dark` from below.

```scss
&--dark {
  background-color: $theme-dark-bg;
  color: $theme-dark-color;

  .go-toast-content--dark {
    color: $theme-dark-color;
  }
}